packages/app/src/containers/ThemeSymbolsInspector.tsx

```diff
@@ -0,0 +1,44 @@
+import { Divider } from 'noya-designsystem';
+import { Selectors } from 'noya-state';
+import { memo, useCallback } from 'react';
+import {
+  useApplicationState,
+  useSelector,
+} from '../contexts/ApplicationStateContext';
+import useShallowArray from '../hooks/useShallowArray';
+import withSeparatorElements from '../utils/withSeparatorElements';
+import NameInspector from '../components/inspector/NameInspector';
+import OpacityInspector from './OpacityInspector';
+import ShadowInspector from './ShadowInspector';
+import { delimitedPath } from 'noya-utils';
+
+export default memo(function ThemeSymbolsInspector() {
+  const [, dispatch] = useApplicationState();
+
+  const selectedSymbols = useShallowArray(
+    useSelector(Selectors.getSelectedSymbols),
+  );
+
+  const handleNameChange = useCallback(
+    (value: string) =>
+      dispatch(
+        'setSymbolName',
+        selectedSymbols.map((v) => v.do_objectID),
+        value,
+      ),
+    [dispatch, selectedSymbols],
+  );
+
+  if (selectedSymbols.length === 0) return null;
+
+  const elements = [
+    <NameInspector
+      names={selectedSymbols.map((v) => delimitedPath.basename(v.name))}
+      onNameChange={handleNameChange}
+    />,
+    <OpacityInspector />,
```

Comment:
The SymbolMaster inspector shouldn't show opacity or shadows:
 
![Pasted_Image_5_18_21__9_24_AM](https://user-images.githubusercontent.com/1198882/118710511-3e87c600-b7d3-11eb-80e6-d0800ca797c1.png)

The SymbolInstance inspector will though, when we get to that!